src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -18,6 +18,13 @@
 
 namespace Azure.Functions.Cli.Common
 {
+    internal enum BackendType
```

Comment:
I plan on either moving this into a separate file or perhaps making it a private inner class.